source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -75,4 +83,5 @@ def get(self, request, instance, shape=None):
     def geo_status(self, instance):
         raise NotImplementedError
 
-
+    def get_coords(self, param):
+        return param.split(";")
```

Comment:
I think it would be nice to add a comment saying that for moment we consider that the param can only be a coordinate